scripts/utils/internals.js

```diff
@@ -171,6 +168,7 @@ module.exports = function (web3 = web3, artifacts = artifacts) {
    * @return {Address} Fallback contract of the input Gnosis Safe
    */
   const getFallbackHandler = async function (safeAddress) {
+    const fallbackHandlerStorageSlot = (await web3.utils.keccak256("fallback_manager.handler.address")).toString("hex")
```

Comment:
nit: the `keccak256` method already returns a string. Also according to the docs it isn't a promise
```suggestion
    const fallbackHandlerStorageSlot = web3.utils.keccak256("fallback_manager.handler.address")
```

Also, this was a module level constant before. Any reason why you moved it? I think this value is fixed and doesn't need to be calculated for each call.

(and to answer your question, I think the `0x` was there before because `ethereumjs-utils` returned a different value than the web3 utils).

Reply:
It needs to be awaited from web3. I moved it in here because it’s only used here and it’s now in an async function. Does that seem appropriate?